data/bash-completion/fwupdmgr.in

```diff
@@ -65,7 +65,7 @@ _fwupdmgr_opts=(
 _show_filters()
 {
 	local flags
-	flags="$(command @libexecdir@/fwupdtool get-device-flags 2>/dev/null)"
```

Comment:
AFAICT this was the only thing being replaced by configuration data.  So perhaps `fwupdmgr.in` should rename to `fwupdmgr` and then no more configuration data processing/replacing.